index.html

```diff
@@ -85,7 +84,35 @@ <h1>I'm Jonathan Glad to see you</h1>
         </div>
     </section>
 </main>
-</div>
-</div>
+<section class="project-section">
+  <div class="project-container">
+      <div class="project-card">
+          <img src="./assets/Snapshoot Portfolio-2.png" alt="project card image" />
+          <div class="prject-card-info flex column">
+              <h2 class="project-title">Tonic</h2>
+              <div class="project-meta flex">
+                  <p class="bold">Canopy</p>
+                  <div class="round"></div>
+                  <p>Backend</p>
+                  <div class="round"></div>
+                  <p>2015</p>
+              </div>
+              <div class="project-note">
+                  <p>
+                      A daily selection of privately personalized reads; no accounts
+                      or sign-ups required.
+                  </p>
+              </div>
+              <div class="project-tag">
+                  <ul class="project-skills flex">
+                      <li>html</li>
+                      <li>css</li>
+                      <li>javascript</li>
+                  </ul>
+              </div>
+              <button class="see-project-btn">See Project</button>
+          </div>
```

Comment:
- [ ] Kindly add more projects to your portfolio as it is in the template.

Reply:
@Kamba56  I don't get your point

Reply:
> @Kamba56 I don't get your point

@KAYEMBAJONATHAN, I can only see one project on your portfolio and I can't see the about section. Kindly add the number of project cards available on the figma template to your portfolio. Also kindly add the about section as this is demanded by the project requirements.